cdap-docs/reference-manual/source/http-restful-api/configuration.rst

```diff
@@ -121,11 +121,6 @@ format, the output would be similar to (showing the first portion, reformatted t
         <value>org.apache.hadoop.yarn.ipc.HadoopYarnProtoRPC</value>
       <source>yarn-default.xml</source>
     </property>
-    <property>
-      <name>mapreduce.job.maxtaskfailures.per.tracker</name>
```

Comment:
i believe this is unrelated to CDAP Tracker.

Reply:
Yep, adding this back in.